lib/questions.rb

```diff
@@ -0,0 +1,11 @@
+class Questions
```

Comment:
Should the tests for this class be in a separate test file?

Reply:
Yes!  
Ideally your spec folder mirrors your lib folder
my_app/
   lib/
       animal.rb
       mammal.rb
       reptile.rb
  Spec/
      animal_spec.rb
      mammal_spec.rb
      reptile_spec.rb

Reply:
👍🏽